sdcm/cluster.py

```diff
@@ -304,12 +304,16 @@ def configure_rsyslog(cls, node, enable_ngrok=False):
     @classmethod
     def get_startup_script(cls):
         post_boot_script = '#!/bin/bash'
-        post_boot_script += dedent(r'''
-               sudo sed -i 's/#MaxSessions \(.*\)$/MaxSessions 1000/' /etc/ssh/sshd_config
```

Comment:
all this configuration is needed, why are you throwing it away ?

Reply:
you need to extend it with the new part handling newer sshd versions, not replace it.

remove this might affect other things, like longevity and such, please put it back.

Reply:
ok. i just noticed this code doesn't exist in branch-2023.1 anymore.

Reply:
fixed, re-added